Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -382,10 +383,35 @@ void PPCDebugInterface::RunToBreakpoint()
 {
 }
 
+std::shared_ptr<Core::NetworkCaptureLogger> PPCDebugInterface::NetworkLogger()
+{
+  const bool has_ssl = Config::Get(Config::MAIN_NETWORK_SSL_DUMP_READ) ||
+                       Config::Get(Config::MAIN_NETWORK_SSL_DUMP_WRITE);
+  const auto current_capture_type =
+      has_ssl ? Core::NetworkCaptureType::Raw : Core::NetworkCaptureType::None;
+
+  if (m_network_logger && m_network_logger->GetCaptureType() != current_capture_type)
+    m_network_logger.reset();
+
+  if (!m_network_logger)
+  {
+    switch (current_capture_type)
+    {
+    case Core::NetworkCaptureType::Raw:
+      m_network_logger = std::make_shared<Core::BinarySSLCaptureLogger>();
+      break;
+    case Core::NetworkCaptureType::None:
+      m_network_logger = std::make_shared<Core::DummyNetworkCaptureLogger>();
```

Comment:
Did you intentionally leave off the final `break` here?

Reply:
Yes, since it's the last case statement. I can add it back if it's preferred.

Reply:
I don't quite see why that is a reason to omit it; I'd prefer (and I think even _we_ prefer, according to the coding guidelines) that every case stament is terminated with either a `break` or an intentional `[[fallthrough]]`. Makes the code look the same everywhere, and isn't anything to worry about if you add more cases or shuffle them around.

Reply:
Done.